Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Header.css';
+import HeaderTitle from './HeaderTitle.js';
+import HeaderContent from './HeaderContent.js';
+import ImageThumbnail from './ImageThumbnail';
+const HeaderContainer = () => {
+    return (
+    <header>
+        <ImageThumbnail />
+        <div className='header-info'>
```

Comment:
I don't see any styling added to the header-info class. If you never need to use a class name to target an element, I would just remove it.